tests/unit/pages/custom_pages/__init__.py

```diff
@@ -0,0 +1,2 @@
+# this directory is used for testing custom values of `pages_folder` param that
+# need to exist as actual paths.
```

Comment:
You're adding a bunch of `__init__.py` files to the tests in this PR - are these now required for pages to work or did you just add them for best practice?

Reply:
In general, no, `__init__.py` are not required for Dash pages to work. However Dash Pages now detects when discovered page modules happen to be inside of a package so this it can add the package name as a prefix to the module path. See `_infer_module_name`. Note this relies on configuring Dash like so: `Dash(name=__package__)`

So in this particular case the `__init__.py` needs to be added to make the test app a package for the tests to work. The other instances I just added them in for best-practice.